app/controllers/conversations.js

```diff
@@ -296,16 +386,23 @@ class ConversationsController extends BaseController {
       getParticipantsByCids: { cids },
     } = data;
 
-    const participantIds = await ConversationParticipant.findAll(
-      { conversation_id: { $in: cids } },
-      ["user_id"],
+    const conversations = await Conversation.findAll(
+      { _id: { $in: cids } },
+      ["participants", "owner_id"],
       null
     );
 
-    const ids = participantIds.map((p) => p.user_id);
+    const usersIds = new Set();
+    conversations.forEach((chat) => {
+      for (const uId of chat.participants) {
```

Comment:
I'm not sure it works. There is no `participants` field in conversation. Does it really work? Do we have a test for it?